conanfile.py

```diff
@@ -281,6 +284,8 @@ def library_pattern(library):
             libs.append(library_pattern('propgrid'))
         if self.options.debugreport:
             libs.append(library_pattern('qa'))
+        if self.options.regex:
+            libs.append('wxregex{unicode}{debug}{suffix}')
```

Comment:
I think something is missing here, like `'wxregex{unicode}{debug}{suffix}'.format(...)`

Reply:
`lib.format()` is used below so I am not yet sure what is causing the build to fail here.